MAINTAINERS.md

```diff
@@ -35,6 +35,10 @@ If you are an owner of the organization, you should be able to see an automated
   [9 May 2019](https://github.com/tldr-pages/tldr/issues/2988) - present
 - **Ein Verne ([@einverne](https://github.com/einverne))**:
   [29 October 2019](https://github.com/tldr-pages/tldr/issues/3488) - present
+- **Proscream ([@Proscream](https://github.com/Proscream))**:
+[22nd November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
+- **Pierre Rudloff ([@Rudloff](https://github.com/Rudloff))**:
+[19th November 2019](https://github.com/tldr-pages/tldr/issues/3592) - present
```

Comment:
We're not using ordinal format for the dates :) please change these to match the existing ones.

Reply:
Also, the links are duplicated; and finally, the entries should be in chronological order.

Reply:
Haha - looks like this old conversation was never resolved.  Resolving it now :)